task/run-script-oci-ta/0.1/run-script-oci-ta.yaml

```diff
@@ -0,0 +1,247 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: run-script
+  annotations:
+    tekton.dev/pipelines.minVersion: 0.12.1
+    tekton.dev/tags: image-build, konflux
+  labels:
+    app.kubernetes.io/version: "0.1"
+spec:
+  description: |-
+    This task allows to run an script stored on the git repository and stores the resultant directory
+    to use it later.
+  params:
+    - name: enableSymlinkCheck
+      description: |
+        Check symlinks in the repo. If they're pointing outside of the repo, the build will fail.
+      type: string
+      default: "true"
+    - name: ociArtifactExpiresAfter
+      description: Expiration date for the trusted artifacts created in the
+        OCI repository. An empty string means the artifacts do not expire.
+      type: string
+      default: ""
+    - name: ociStorage
+      description: The OCI repository where the Trusted Artifacts are stored.
+      type: string
+    - name: SCRIPT_RUNNER_IMAGE
+      description: The image to run the script in
+      type: string
+    - name: SCRIPT
+      description: The script to launch
+      type: string
+    - name: SCRIPT_WORKDIR
+      description: The directory to launch the script from
+      type: string
+      default: /var/workdir/script
+    - name: SCRIPT_ARTIFACT_DIRECTORY
+      description: The directory to include on the output Trusted Artifact
+      type: string
+      default: /var/workdir/script
+    - name: STORAGE_DRIVER
+      description: Storage driver to configure for buildah
+      type: string
+      default: vfs
+    - name: HERMETIC
+      description: Determines if build will be executed without network access.
+      type: string
+      default: "false"
+    - name: SOURCE_ARTIFACT
+      description: The Trusted Artifact URI pointing to the artifact with
+        the application source code.
+      type: string
+    - name: caTrustConfigMapKey
+      description: The name of the key in the ConfigMap that contains the
+        CA bundle data.
+      type: string
+      default: ca-bundle.crt
+    - name: caTrustConfigMapName
+      description: The name of the ConfigMap to read CA bundle data from.
+      type: string
+      default: trusted-ca
+  results:
+    - name: SCRIPT_ARTIFACT
+      description: The Trusted Artifact URI pointing to the artifact with
+        the application source code.
+    - name: SCRIPT_BASE_IMAGE
+      description: Reference and digest of the image used to run the script. Suitable to pass to buildah ADDITIONAL_BASE_IMAGES
+      type: string
+    - name: SCRIPT_OUTPUT
```

Comment:
Here I'm the one diverging from the requirements doc. The doc defines these as `OUTPUT` and `OUTPUT_ARRAY`. However, the rest of the parameter and results related to the script execution are previxed with `SCRIPT`.

Should I keep prefixed and we should update the doc, or should I rename them to remove the prefix?

Reply:
It seems like it would be better to consistently name these results.

Reply:
now that we moved to use `SCRIPT_RUNNER_IMAGE` as the input param, I don't like at all the `SCRIPT_BASE_IMAGE` as the result var name either :joy:

Reply:
renamed it to `SCRIPT_RUNNER_IMAGE_REFERENCE` . Now everything is consistent and is prefixed with `SCRIPT_` . Are we good here now? :)